Sammuel09/Fast-Food-Fast
# #160704349 API endpoints to get all menu (users)

#### What does this PR do?
Allows a user to get all menu options

#### Description of Task to be completed?
- Users can get all menu options
- The route is authenticated such that users need to be logged in or signed up to access the route.

### How should this be manually tested?
First, signup to the app by making a POST request to https://sammie-fast-food-fast.herokuapp.com/api/v1/auth/signup

Use data in the following format:
`username: string`,
`email: string`,
`phone: string`,
`address: string`,
`password: string`

Copy the token received in the response in the signup process and add it as part of the request headers in this format:
`x-access-token: token`

On Postman, visit https://sammie-fast-food-fast.herokuapp.com/api/v1/menu
Then make a GET Request 
Expected Data to be returned:
`status: string`,
`data: array`,
`message: string`

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/n/projects/2196965/stories/160704349
